recipes/python-bioext/activate.sh

```diff
@@ -0,0 +1 @@
+export BEALIGN_REFERENCE_DIR=usr/local/share/bealign_references
```

Comment:
This path need to be updated. It's probably easier to generate this file directly from `build.sh`